_nodes/0021.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Opcartesian morphisms
```

Comment:
can we call it cocartesian instead?

Reply:
And cocartesian fibration? Sure.

Reply:
Well it's a tiny bit awkward because people shy away from "cofibration" for obvious reasons, but I don't feel strongly.

Reply:
right. i don’t want to say cofibration. i think it is just very common in this school to say cocartesian.

Reply:
Indeed, I just mean that "cocartesian fibration" is more different to "opfibration" than "opcartesian fibration" is, but I don't mind for any reason other than potential confusion.